<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow2</class>
 <widget class="QMainWindow" name="MainWindow2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>690</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Управление ботом РУДН</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>8</y>
      <width>791</width>
      <height>681</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="l_name">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>MS Shell Dlg 2</family>
         <pointsize>14</pointsize>
         <weight>9</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>Привет, пользователь!</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="currentIndex">
        <number>2</number>
       </property>
       <widget class="QWidget" name="tab_1">
        <attribute name="title">
         <string>Обычное расписание</string>
        </attribute>
        <widget class="QPushButton" name="b_thu_1">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>310</y>
           <width>170</width>
           <height>100</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Четверг
(нечётная неделя)</string>
         </property>
        </widget>
        <widget class="QPushButton" name="b_tue_0">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>110</y>
           <width>170</width>
           <height>100</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
         </property>
         <property name="text">
          <string>Вторник
(чётная неделя)</string>
         </property>
        </widget>
        <widget class="QPushButton" name="b_fri_1">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>410</y>
           <width>170</width>
           <height>100</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Пятница
(нечётная неделя)</string>
         </property>
        </widget>
        <widget class="QPushButton" name="b_fri_0">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>410</y>
           <width>170</width>
           <height>100</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;</string>
         </property>
         <property name="text">
          <string>Пятница
(чётная неделя)</string>
         </property>
        </widget>
        <widget class="QPushButton" name="b_sat_1">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>510</y>
           <width>170</width>
           <height>100</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Суббота
(нечётная неделя)</string>
         </property>
        </widget>
        <widget class="QPushButton" name="b_tue_1">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>110</y>
           <width>170</width>
           <height>100</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Вторник
(нечётная неделя)</string>
         </property>
        </widget>
        <widget class="QPushButton" name="b_wed_0">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>210</y>
           <width>170</width>
           <height>100</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
         </property>
         <property name="text">
          <string>Среда
(чётная неделя)</string>
         </property>
        </widget>
        <widget class="QPushButton" name="b_mon_0">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>10</y>
           <width>170</width>
           <height>100</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
         </property>
         <property name="text">
          <string>Понедельник
(чётная неделя)</string>
         </property>
        </widget>
        <widget class="QPushButton" name="b_wed_1">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>210</y>
           <width>170</width>
           <height>100</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Среда
(нечётная неделя)</string>
         </property>
        </widget>
        <widget class="QPushButton" name="b_mon_1">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>10</y>
           <width>170</width>
           <height>100</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Понедельник
(нечётная неделя)</string>
         </property>
        </widget>
        <widget class="QPushButton" name="b_sat_0">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>510</y>
           <width>170</width>
           <height>100</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;</string>
         </property>
         <property name="text">
          <string>Суббота
(чётная неделя)</string>
         </property>
        </widget>
        <widget class="QPushButton" name="b_thu_0">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>310</y>
           <width>170</width>
           <height>100</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;</string>
         </property>
         <property name="text">
          <string>Четверг
(чётная неделя)</string>
         </property>
        </widget>
        <widget class="QPlainTextEdit" name="plainTextEdit">
         <property name="geometry">
          <rect>
           <x>350</x>
           <y>80</y>
           <width>431</width>
           <height>451</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="clear_PTE">
         <property name="geometry">
          <rect>
           <x>349</x>
           <y>575</y>
           <width>211</width>
           <height>35</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 125, 125);
font: 75 14pt &quot;MS Shell Dlg 2&quot;;</string>
         </property>
         <property name="text">
          <string>Сбросить изменения</string>
         </property>
        </widget>
        <widget class="QPushButton" name="send_changes">
         <property name="geometry">
          <rect>
           <x>560</x>
           <y>575</y>
           <width>220</width>
           <height>35</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(125, 255, 125);
font: 75 14pt &quot;MS Shell Dlg 2&quot;;</string>
         </property>
         <property name="text">
          <string>Применить изменения</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>350</x>
           <y>10</y>
           <width>160</width>
           <height>30</height>
          </rect>
         </property>
         <property name="text">
          <string>Выбранный день:</string>
         </property>
        </widget>
        <widget class="QLabel" name="l_day">
         <property name="geometry">
          <rect>
           <x>510</x>
           <y>10</y>
           <width>271</width>
           <height>30</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>350</x>
           <y>540</y>
           <width>170</width>
           <height>30</height>
          </rect>
         </property>
         <property name="text">
          <string>Статус изменения:</string>
         </property>
        </widget>
        <widget class="QLabel" name="l_status">
         <property name="geometry">
          <rect>
           <x>520</x>
           <y>540</y>
           <width>250</width>
           <height>30</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="label_7">
         <property name="geometry">
          <rect>
           <x>350</x>
           <y>50</y>
           <width>241</width>
           <height>30</height>
          </rect>
         </property>
         <property name="text">
          <string>Введите текст сообщения:</string>
         </property>
        </widget>
        <widget class="Line" name="line">
         <property name="geometry">
          <rect>
           <x>350</x>
           <y>40</y>
           <width>431</width>
           <height>20</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Временные изменения</string>
        </attribute>
        <widget class="QPlainTextEdit" name="PTE_temp">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>300</y>
           <width>650</width>
           <height>171</height>
          </rect>
         </property>
        </widget>
        <widget class="QDateEdit" name="DE_temp">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>510</y>
           <width>771</width>
           <height>51</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="clear_PTE_temp">
         <property name="geometry">
          <rect>
           <x>660</x>
           <y>390</y>
           <width>120</width>
           <height>80</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 125, 125);
font: 75 14pt &quot;MS Shell Dlg 2&quot;;</string>
         </property>
         <property name="text">
          <string>Сбросить
текст
сообщения</string>
         </property>
        </widget>
        <widget class="QPushButton" name="send_chages_temp">
         <property name="geometry">
          <rect>
           <x>660</x>
           <y>300</y>
           <width>120</width>
           <height>80</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(125, 255, 125);
font: 75 14pt &quot;MS Shell Dlg 2&quot;;</string>
         </property>
         <property name="text">
          <string>Отправить
текст
сообщения</string>
         </property>
        </widget>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>570</y>
           <width>171</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Статус изменения:</string>
         </property>
        </widget>
        <widget class="QLabel" name="l_status_2">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>565</y>
           <width>250</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>270</y>
           <width>251</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Введите текст сообщения:</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>10</y>
           <width>250</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Запланированные события:</string>
         </property>
        </widget>
        <widget class="QPushButton" name="delete_temp_note">
         <property name="geometry">
          <rect>
           <x>660</x>
           <y>30</y>
           <width>120</width>
           <height>80</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 125, 125);
font: 75 14pt &quot;MS Shell Dlg 2&quot;;</string>
         </property>
         <property name="text">
          <string>Удалить
запись</string>
         </property>
        </widget>
        <widget class="QTableWidget" name="TW_temp">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>30</y>
           <width>651</width>
           <height>241</height>
          </rect>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QAbstractScrollArea::AdjustToContents</enum>
         </property>
         <property name="rowCount">
          <number>4</number>
         </property>
         <attribute name="horizontalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderCascadingSectionResizes">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>250</number>
         </attribute>
         <attribute name="horizontalHeaderHighlightSections">
          <bool>true</bool>
         </attribute>
         <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
          <bool>true</bool>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderCascadingSectionResizes">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderDefaultSectionSize">
          <number>50</number>
         </attribute>
         <attribute name="verticalHeaderHighlightSections">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderShowSortIndicator" stdset="0">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderStretchLastSection">
          <bool>false</bool>
         </attribute>
         <row/>
         <row/>
         <row/>
         <row/>
         <column>
          <property name="text">
           <string>Текст сообщения</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Дата сообщения</string>
          </property>
         </column>
        </widget>
        <widget class="QLabel" name="label_8">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>470</y>
           <width>251</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Выберите дату сообщения:</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_3">
        <attribute name="title">
         <string>Список студентов</string>
        </attribute>
        <widget class="QTableWidget" name="tableWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>761</width>
           <height>591</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>761</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>761</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QAbstractScrollArea::AdjustToContents</enum>
         </property>
         <property name="rowCount">
          <number>23</number>
         </property>
         <attribute name="horizontalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderCascadingSectionResizes">
          <bool>true</bool>
         </attribute>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>150</number>
         </attribute>
         <attribute name="horizontalHeaderHighlightSections">
          <bool>true</bool>
         </attribute>
         <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderCascadingSectionResizes">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderDefaultSectionSize">
          <number>30</number>
         </attribute>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <column>
          <property name="text">
           <string>Фамилия</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Имя</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Отчество</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Номер студенческого</string>
          </property>
         </column>
         <item row="0" column="0">
          <property name="text">
           <string>Кочетков</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </item>
         <item row="0" column="1">
          <property name="text">
           <string>Максим</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </item>
         <item row="0" column="2">
          <property name="text">
           <string>Владимирович</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </item>
         <item row="0" column="3">
          <property name="text">
           <string>58261561651</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </item>
         <item row="1" column="0">
          <property name="text">
           <string/>
          </property>
         </item>
         <item row="1" column="3">
          <property name="text">
           <string/>
          </property>
          <property name="flags">
           <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
          </property>
         </item>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
